src/tests/JIT/Methodical/tailcall/test_implicit.il

```diff
@@ -80,6 +84,7 @@ ret
     IL_0049:  br.s       IL_004b
 
     IL_004b:  ldloc.0
+exit:
```

Comment:
Is this new label unreferenced?

Reply:
Hmm, you're right, I have no idea why I put it there. Thanks for spotting. will remove.

Reply:
Fixed in 2nd commit.